solutions/usaco-810.mdx

```diff
@@ -161,4 +161,26 @@ public class RestStops {
 
 </JavaSection>
 
+<PySection>
+
+```python
+with open('reststops.in') as read:
+	l, n, f, b = map(int, read.readline().split())
+	
+	x, c = [0] * (n + 1), [0] * (n + 1)
+	
+	for i in range(1, n + 1):
+		x[i], c[i] = map(int, read.readline().split())
+
+ans = 0
+
+for i in range(n, 0, -1):
+	c[i] = max(c[i], c[i + 1])
```

Comment:
list index out of range lol

Reply:
no way let me check... all the solutions should work (they worked in the beginning)

Reply:
@envyaims its not bc the size of c is n + 1 (refer to line 170)

Reply:
try submitting it

Reply:
oo changed it should work now @/ whoever told me change my solution before u were wrong ig.